spacy_llm/tests/pipeline/test_llm.py

```diff
@@ -234,6 +237,30 @@ def test_llm_logs_default_null_handler(nlp: Language, capsys: pytest.CaptureFixt
     assert f"LLM response for doc: {doc.text}" not in captured.out
 
 
+def test_fewshot_reader():
```

Comment:
This seems to be a specific test when you save a `yaml` file with wrong suffixes. Should there be a different name for the test to reflect that?

Reply:
Updated in https://github.com/explosion/spacy-llm/pull/184/commits/8705b538d563eba28e1226f5f5a814ad4da5eb09. Better?